api/src/services/__tests__/competenciesService.ts

```diff
@@ -0,0 +1,22 @@
+import { listCompetencies } from '../competenciesService';
+import { mockQuery } from '../mockDb';
+
+describe('listCompetencies', () => {
+  it('should query the lists of the competencies in the database with their titles and descriptions.', async () => {
```

Comment:
This test description states that "titles" are queried for, but the test doesn't have anything in it called "title". Is this supposed to be "labels"?

Also, test descriptions typically do not have a period at the end.

Reply:
Yes, it was supposed to be "labels". We changed it now to 'labels'. Also removed full stop.